bazel/wasm.bzl

```diff
@@ -42,18 +42,28 @@ wasi_rust_transition = transition(
 
 def _wasm_binary_impl(ctx):
     out = ctx.actions.declare_file(ctx.label.name)
-    ctx.actions.run(
-        executable = "cp",
-        arguments = [ctx.files.binary[0].path, out.path],
-        outputs = [out],
-        inputs = ctx.files.binary,
-    )
+    if ctx.attr.signing_key:
+        ctx.actions.run(
+            executable = ctx.executable._wasmsign_tool,
+            arguments = ["--sign", "--use-custom-section", "--sk-path", ctx.files.signing_key[0].path, "--pk-path", ctx.files.signing_key[1].path, "--input", ctx.files.binary[0].path, "--output", out.path],
+            outputs = [out],
+            inputs = ctx.files.binary + ctx.files.signing_key,
+        )
+    else:
+        ctx.actions.run(
+            executable = "cp",
+            arguments = [ctx.files.binary[0].path, out.path],
+            outputs = [out],
+            inputs = ctx.files.binary,
+        )
 
     return [DefaultInfo(files = depset([out]), runfiles = ctx.runfiles([out]))]
 
 def _wasm_attrs(transition):
     return {
         "binary": attr.label(mandatory = True, cfg = transition),
+        "signing_key": attr.label_list(allow_files = True),
+        "_wasmsign_tool": attr.label(default = "@proxy_wasm_cpp_host__wasmsign__0_1_2//:cargo_bin_wasmsign", executable = True, cfg = "exec"),
```

Comment:
wondering if there is an option in cargo-raze to have the alias for `cargo_bin_wasmsign`..

Reply:
Done.